index.html

```diff
@@ -0,0 +1,19 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="utf-8"/>
+    <title>Renoma</title>
+    <link rel="stylesheet" type="text/css" href="style.css">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+</head>
+<body>
+    <header>
+        <nav>
+            <div class="Hamburger"><a><img src="images/Hamburger.png" alt="Hamburger butten image"></a></div>
```

Comment:
Change, please `<div class="Hamburger">` on `<div class="hamburger__wrapper">` according to [BEM ](https://css-tricks.com/bem-101/) and for img set `<img class="hamburger" src="images/Hamburger.png" alt="Hamburger butten image">` 
Also please remove the tag `<a>` it doesn't need. In the case when need open something by click need to use _javascript_, but at this moment, it's not in the scope of the current task.